config.yml

```diff
@@ -10,7 +10,6 @@ params:
     readmore: true
     opengraph: true
     show_stream: true
-    competition_livestream_url: https://www.facebook.com/SUSUtv/videos/vb.159049780794219/1891818877517292/?type=3&theater
```

Comment:
What's the reason for removing this? It won't actually get used anywhere, but does act as documentation for what the value should be.

Reply:
Mostly cleanup

Reply:
Ok, I don't think we should omit config values which are still present in the site, even if those values don't get used.

Reply:
Agreed on removing the partial. But I still think we should remove the related variables too. The same embed code is used for the competition blog page, and the same video URL, so we can pull it out of that if necessary?

Reply:
6c2b7c9 removes the partial, configs and related styling.